muzlag/internal/cmd/main.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
cmd should not be part of the internal package

https://github.com/golang-standards/project-layout?tab=readme-ov-file#go-directories
https://go.dev/doc/modules/layout